Tests/Functional/Drivers/Queue/AbstractQueueDriverTest.php

```diff
@@ -9,6 +9,8 @@
 use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\PurgeableQueueDriverInterface;
 use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\QueueDriverInterface;
 use Smartbox\Integration\FrameworkBundle\Components\Queues\QueueMessage;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\QueueProtocol;
+use Smartbox\Integration\FrameworkBundle\Core\Consumers\ConsumerInterface;
```

Comment:
I don't see any use of these 2 class in this class test, so I guess it can be deleted

Reply:
Ok. I removed.